### PR TITLE
Independent testing

### PR DESCRIPTION
**Pull request by wangben718**
Test for several functions.
Hi Xuan @XDeng0921, please review function and validation for tlf_ae_summary() based on this update.

### CODE REVIEW

**Comment by XDeng0921** [comment]
@wangben718 
Hi Ben, when I review tlf_ae_summary, I found the output is different than the standard AE summary table. In standard, we usually have the below two rows as fixed, with other customized AE followed. 

Capture

Not sure whether we discussed this before, but do we want to have the first 2 rows as default?

**Review by LittleBeannie — COMMENTED** [review]
Hi Ben, I have 2 minor comments, please take a look.

File: R/tlf_ae_summary.R [hunk 1]
```diff
@@ -110,6 +110,12 @@ tlf_ae_summary <- function(population_from,
     res$tot_n <- res$n_1 + res$n_2
   }
   
+  if(!is.null(ae_interested)){
+    if(tolower(ae_interested)=="null"){
```
Comment (wangben718) [inline R/tlf_ae_summary.R:114]: HI Yujie @LittleBeannie : I tried to used your code for ae_interested = "null", however it doesn't work. as.null() would convert all criterion into NULL. Maybe we could have a better way to do this in the future.

File: R/tlf_ae_summary.R [hunk 2]
```diff
@@ -110,17 +110,15 @@ tlf_ae_summary <- function(population_from,
     res$tot_n <- res$n_1 + res$n_2
   }
   
-  if(!is.null(ae_interested)){
-    if(tolower(ae_interested)=="null"){
-      ae_interested <- as.null(ae_interested)
-    }
-  }
+  interested_ae_label_1 <- ifelse("null" %in% tolower(ae_interested$interested_ae_criterion),
```
Comment (wangben718) [inline R/tlf_ae_summary.R:113]: Hi Yujie @LittleBeannie : Could you please review my change? Is it what you want? Thanks

Comment (LittleBeannie) [reply R/tlf_ae_summary.R:113]: Yes! Please continue work on the CMD check.

**Pull request merged**